source/Halibut.Tests/Support/TestAttributes/LatestClientAndLatestServiceTestCasesAttribute.cs

```diff
@@ -23,18 +23,19 @@ public class LatestClientAndLatestServiceTestCasesAttribute : TestCaseSourceAttr
             bool testSyncClients = true,
             bool testAsyncClients = true,
             bool testAsyncServicesAsWell = false, // False means only the sync service will be tested.
+            bool testSyncService = true,
```

Comment:
I wonder if these could be in a `#define` and so defined once but used here and in the below static method. As well as in other builders.